src/components/EventsTimeline/TimelineEventItem.tsx

```diff
@@ -18,13 +19,20 @@ type Props = {
   itemCount?: number
 }
 
+type ItemType = {
+  label: string
+  value: string | boolean
+  value_verbose: string
+}
+
 const TimelineEventItem: React.FC<Props> = ({ fields, caseEvents, title = "", dateField = "date_created", pathName, isOpen = false, useTransparentBackground = false, itemCount = 0 }) => {
 
   // This situation would be considered a problem within the data returned from the API
   if (caseEvents.length === 0) return null
```

Comment:
Think you probably want to move line 35 above here. And check `if (caseEvents.length === 0 && items.length === 0)`.

Reply:
Not really. `Items` is only used for the fields in `variables`. Not every `caseEvent` has variables with fields in it but still has to be shown (for instance every static form)